src/pages/Rules/Rules.jsx

```diff
@@ -0,0 +1,47 @@
+import "../../css/main.css";
+import "./Rules.css";
+import { useEffect } from "react";
+import { Link, useParams } from "react-router-dom";
+import { useScrollToTop, useDocumentTitle } from "../../utils";
+import { RulesData } from "./RulesData";
+import { useQuiz } from "../../context";
+import { Loader } from "../../components";
+
+export const Rules = () => {
+  useScrollToTop();
+  useDocumentTitle("Cric Quiz | Rules");
+  const { quizId } = useParams();
+  const { getCategoryData, selectedQuiz } = useQuiz();
+  const { selectedQuizLoading } = selectedQuiz;
+
+  useEffect(() => {
+    getCategoryData(quizId);
+  }, [quizId]);
+
+  return (
+    <main className="flex-grow-1">
+      {selectedQuizLoading && <Loader />}
+      {!selectedQuizLoading && (
```

Comment:
One suggestion is that instead of checking `selectedQuizLoading` twice with `&&`, you can use ternary operator ` ? :`

Reply:
Yes I was doing the same previously but I read one article suggesting not to use ternary operator in jsx. The main intention is to increase the readability of the code is what that article suggested about.